projects/robots/clearpath/heron/worlds/ocean.wbt

```diff
@@ -0,0 +1,120 @@
+#VRML_SIM R2024a utf8
+
+EXTERNPROTO "https://raw.githubusercontent.com/cyberbotics/webots/R2024a/projects/objects/backgrounds/protos/TexturedBackground.proto"
+EXTERNPROTO "https://raw.githubusercontent.com/cyberbotics/webots/R2024a/projects/objects/backgrounds/protos/TexturedBackgroundLight.proto"
+EXTERNPROTO "https://raw.githubusercontent.com/cyberbotics/webots/R2024a/projects/objects/obstacles/protos/OilBarrel.proto"
+EXTERNPROTO "../protos/Heron.proto"
+
+WorldInfo {
+}
+Viewpoint {
+  orientation -0.24219658556528326 0.039505018164956274 0.9694225948884753 2.986262471610857
+  position 6.727587480647112 -2.788009572581076 8.707663910740424
+}
+TexturedBackground {
+}
+TexturedBackgroundLight {
+}
+DEF WATER Fluid {
+  children [
+    Shape {
+      appearance PBRAppearance {
+        baseColor 0.0626841 0.552224 0.723339
+        transparency 0.7
+        roughness 1
+        metalness 0
+      }
+      geometry DEF WATER_BOX Box {
+        size 1000 1000 0.7
+      }
+    }
+  ]
+  viscosity 0.01
+  boundingObject Transform {
+    children [
+      USE WATER_BOX
+    ]
+  }
+  locked TRUE
+}
+Heron {
+  linearVelocity_0 7.738948701030327e-55 -4.450876199231589e-55 2.819335278757325e-06
+  angularVelocity_0 -0.0010746087365596433 -0.0018872241483721332 -8.299049855187801e-20
```

Comment:
Please remove also all these linearVelocity_0 and angularVelocity_0 parameters which are useless.

Reply:
I believe you should really remove these lines, not replace them.
Also, please update the icon.